### PR TITLE
Expanded AspNet test coverage

### PR DESCRIPTION
**Pull request by CodeBlanch**
Expanded the AspNet HttpInListener tests to cover the bug & fix from #1376.

### CODE REVIEW

**Review by reyang — APPROVED** [review]
LGTM.

File: test/OpenTelemetry.Instrumentation.AspNet.Tests/HttpInListenerTests.cs [hunk 1]
```diff
@@ -132,11 +133,16 @@ public void Dispose()
...
                     expectedTraceId,
                     expectedSpanId,
-                    ActivityTraceFlags.Recorded),
+                    ActivityTraceFlags.Recorded,
+                    isRemote: true),
                 default));
 
-            var activity = new Activity(ActivityNameAspNet).AddBaggage("Stuff", "123");
-            activity.SetParentId(expectedTraceId, expectedSpanId, ActivityTraceFlags.Recorded);
+            var activity = new Activity(HttpInListener.ActivityOperationName).AddBaggage("Stuff", "123");
```
Comment (cijothomas) [inline test/OpenTelemetry.Instrumentation.AspNet.Tests/HttpInListenerTests.cs:140]: wondering what is the relevance of this AddBaggage? (its not newly added in this pr anyway)

**Review by cijothomas — APPROVED** [review]

File: test/OpenTelemetry.Instrumentation.AspNet.Tests/HttpInListenerTests.cs (hunk 1 above)
Comment (CodeBlanch) [reply test/OpenTelemetry.Instrumentation.AspNet.Tests/HttpInListenerTests.cs:140]: 🤷 There were those few days where we had the Baggage API built on top of Activity.Baggage, maybe this is a carry-over from then? I don't think it has any relevance to these tests at the moment.

Comment (cijothomas) [reply test/OpenTelemetry.Instrumentation.AspNet.Tests/HttpInListenerTests.cs:140]: Got it. Will cleanup as part of overall cleanups I plan to do in next few days!

**Pull request merged**